migrations/sqls/20211118092400-second-part--charge-transaction-flag-down.sql

```diff
@@ -0,0 +1,11 @@
+alter table water.billing_transactions 
+drop column is_two_part_second_part_charge;
+
+
+alter table water.billing_transactions 
+add column is_two_part_tariff_supplementary boolean default false not null;
+
+
+update water.billing_transactions 
+set is_two_part_tariff_supplementary = true
+where lower(description) like 'second%';
```

Comment:
this is absolutely fine but you could also do:

```
where description ilike 'second%';
```

`ilike` does a case-insensitive comparison.

Reply:
ooo nice one thanks didn't know this